src/components/scenes/CoinRankingScene.tsx

```diff
@@ -54,8 +98,15 @@ const CoinRankingComponent = (props: Props) => {
   const styles = getStyles(theme)
   const { navigation } = props
 
-  const defaultIsoFiat = useSelector(state => `iso:${getDefaultFiat(state)}`)
-  const [lastUsedFiat, setLastUsedFiat] = useState<string>(defaultIsoFiat)
+  /** The user's fiat setting, falling back to USD if CoinGecko does not support
+   * it. */
+  const supportedFiatSetting = useSelector(state => {
```

Comment:
Get into the habit of using lightweight selectors since all selectors run on ALL redux changes. This should just be

```
const defaultFiat = useSelector(state => getDefaultFiat(state))
const supportedFiatSetting = COINGECKO_SUPPORTED_FIATS.includes(defaultFiat) ? defaultFiat : 'USD'
```